en/02_Developer_Guides/05_Extending/05_Injector.md

```diff
@@ -334,17 +407,24 @@ Calling..
 use App\MyController;
 use SilverStripe\Core\Injector\Injector;
 
-// sets up MyController as a singleton
 $controller = Injector::inst()->get(MyController::class);
 ```
 
-Would setup the following
-
-* Create an object of type `App\MyController`
-* Look through the **dependencies** and call `get('App\PermissionService')`
-* Load the configuration for PermissionService, and create an object of type `App\RestrictivePermissionService`
-* Look at the properties to be injected and look for the config for `App\MySQLDatabase`
-* Create a `App\MySQLDatabase` class, passing `dbusername` and `dbpassword` as the parameters to the constructor.
+Would perform the following steps:
+
+* Fetches a singleton of the `App\MyController` service
+* Assuming this is the first time a `App\MyController` singleton is being instantiated:
+  * Instantiates the service as an object of the `App\MyController` class since no other was defined
+  * Look through the `dependencies` for the `App\MyController` service and fetch a singleton of the `App\PermissionService` service
+  * Assuming this is the first time a `App\PermissionService` singleton is being instantiated:
+    * Instantiates the service as an object of the `App\RestrictivePermissionService` class since that class was defined
+    * Look at the properties to be injected for the `App\PermissionService` service fetch a singleton of the `App\MySQLDatabase` service
+    * Assuming this is the first time a `App\MySQLDatabase` singleton is being instantiated:
+      * Instantiates the service as an object of the `App\MySQLDatabase` class since no other was defined
+        * Evaluates and passes in the values of the constants or environment variables `dbusername` and `dbpassword` as arguments to the constructor
```

Comment:
Indentation

Reply:
What do you want to be changed about the indentation?
Note that I have intentionally indented these bullet points - e.g. "assuming this is the first time it's instantiated" applies to the indented items below that. Once the indent is finished, we no longer require that assumption for the rest of the steps.

Reply:
The `Evaluates and passes in the values ` line is what I was referring to, seems like it shouldn't indented